### PR TITLE
feat(widget-input-form): apply reference handler for autocomplete api

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

SSIA

### CODE REVIEW

File: src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/reference-handler-helpers.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        if (!axios.isCancel(e.axiosError)) {
+            ErrorHandler.handleError(e);
+        }
+
+        return [];
+    }
+};
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const getReferenceHandler = (): ReferenceHandler => async (inputText, schema, pageStart, pageLimit) => {
+    console.debug('schema', schema);
```
Comment (piggggggggy) [inline src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/reference-handler-helpers.ts]: question: Maybe you missed `console.log`?

File: src/services/dashboards/widgets/_configs/widget-schema-config.ts [hunk 2]
```diff
@@ -30,27 +30,42 @@ export const COST_REFERENCE_SCHEMA = {
...
+        },
     },
     [GROUP_BY.ACCOUNT]: {
         title: 'Account ID',
         type: 'array',
-        reference: 'cost_analysis.Cost',
+        reference: {
+            resource_type: 'cost_analysis.Cost',
+            reference_key: GROUP_BY.ACCOUNT,
+        },
```
Comment (piggggggggy) [inline src/services/dashboards/widgets/_configs/widget-schema-config.ts:68]: 🥹👍

**Review by piggggggggy — APPROVED** [review]
LGTM!!
I left a little comment!

File: src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/reference-handler-helpers.ts (hunk 1 above)
Comment (WANZARGEN) [reply src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/reference-handler-helpers.ts]: Yes thanks I removed it :)

**Pull request merged**